src/test/java/com/epam/prejap/teatrees/game/PrinterTest.java

```diff
@@ -0,0 +1,58 @@
+package com.epam.prejap.teatrees.game;
+
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+
+public class PrinterTest {
+    private static PrintStream originalPrintStream;
+    private static PrintStream capturedPrintStream;
+    private static ByteArrayOutputStream output;
+
+    @BeforeMethod
+    private void setupStreams() {
+        originalPrintStream = System.out;
+        output = new ByteArrayOutputStream();
+        capturedPrintStream = new PrintStream(output);
+        System.setOut(capturedPrintStream);
```

Comment:
Capturing `System.out` just to test a single class (which, btw, accepts an output stream as a dependency) makes it impossible to run the tests in parallel, as multiple tests capturing System.out will interfere with each other and cause random failures. Instead, pass the print stream to the Printer itself.